nemo/collections/asr/models/hybrid_rnnt_ctc_models.py

```diff
@@ -95,28 +95,28 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
     @torch.no_grad()
     def transcribe(
         self,
-        paths2audio_files: List[str],
+        audio: List[str],
         batch_size: int = 4,
         return_hypotheses: bool = False,
         partial_hypothesis: Optional[List['Hypothesis']] = None,
         num_workers: int = 0,
         channel_selector: Optional[ChannelSelectorType] = None,
         augmentor: DictConfig = None,
         verbose: bool = True,
-        logprobs: bool = False,
+        # logprobs: bool = False, DEPRECATED?
```

Comment:
lets start adding deprecate warnings for 2.0 release

Reply:
Ok but this PR is removing them all together. Reverting this and making it a warning will be too many changes. Should we merge this after Eric cuts r1.23 branch ? Then I can add a deprecation warning to just the r1.23 branchc

Reply:
Definitely, just add depreciation for 1.23 and this PR will go for 2.0 release !  
This is one of major updates to 2.0, lets keep it for 2.0